### ISSUE TITLE
Attach pre-built binaries to github releases

### ISSUE THREAD

**Issue by wpank** [centrifuge/centrifuge-chain#550]
Similar to how Parity does this for Polkadot, would be very useful for node operators to attach pre-built binaries to github releases, as well as include rust versions that are used to compile them.

**Comment by mustermeiszer** [comment]
I think this is a good idea. WDYT @mikiquantum?

**Comment by mustermeiszer** [comment]
With which rust version are you compiling @wpank? Have not built it in a while but this one should work `nightly-2020-08-16`.

**Comment by wpank** [comment]
> With which rust version are you compiling @wpank? Have not built it in a while but this one should work `nightly-2020-08-16`.

Yes, I was eventualy able to compile it with `nightly-2020-10-01`

**Comment by mikiquantum** [comment]
This is a no overall improvement haul on our CI release, we will track it there: https://github.com/centrifuge/centrifuge-chain/issues/400
